test/user_test.rb

```diff
@@ -0,0 +1,56 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/joke'
+require './lib/user'
+
+class UserTest < Minitest::Test
+
+  def test_it_exists
```

Comment:
When we test that it exists, we want to test that the object is indeed an instance of the class we expect it to be. This tests that a `User` object starts off with an empty array of `jokes`. This is a good test, but it needs to be named differently.